types/react-window/index.d.ts

```diff
@@ -0,0 +1,404 @@
+// Type definitions for react-window 1.1
+// Project: http://react-window.now.sh/
```

Comment:
It's better to use the project url https://github.com/bvaughn/react-window rather than doc url